topics/algolia/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: algoliasearch, algolia-search, instantsearch
+created_by: Nicolas Dessaigne, Julien Lemoine
+display_name: Algolia
+github_url: https://github.com/algolia/
+logo: algolia.png
+released: October 23, 2012
+short_description: Algolia is a hosted search API for web and mobile applications
+topic: algolia
+url: https://www.algolia.com/
+wikipedia_url: https://en.wikipedia.org/wiki/Algolia
+---
+Algolia is a hosted API for building search into web and mobile applications, with typo-tolerance, fully configurable relevance and other tools for making great search experiences
```

Comment:
Could you end with a period here and on line 8, in the `short_description`?

I think we also want a comma here after `fully configurable relevance`.

Reply:
sure